### PR TITLE
[deckhouse] Start debug HTTP API server next to socket server.

### PR DESCRIPTION
**Pull request by Dmitrykob**
## Description

Deckhouse has an API that provides info on the state of its internal storage, queues, preflight patches, and more.
This API is now exposed via the Unix socket and can be accessed with the deckhouse-controller CLI.

## Why do we need it, and what problem does it solve?

According to issue https://github.com/deckhouse/deckhouse/issues/4527 we add http server next to socket server to expose api for ui.

## What is the expected result?

The debug api is available via the http protocol. The cube-rback-proxy is responsible for access control.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse
type: feature
summary:  Start debug HTTP API server next to socket server.
impact_level: default
```

### CODE REVIEW

File: modules/002-deckhouse/templates/deployment.yaml [hunk 1]
```diff
@@ -175,6 +176,8 @@ spec:
               value: /tmp/shell-operator-debug.socket
             - name: HISTFILE
               value: /tmp/.bash_history
+            - name: DEBUG_HTTP_PORT
+              value: "9652"
```
Comment (yalosev) [inline modules/002-deckhouse/templates/deployment.yaml:180]: We also have to add this variable [here](https://github.com/deckhouse/deckhouse/blob/main/dhctl/pkg/kubernetes/actions/manifests/manifests.go#L320)

**Comment by Dmitrykob** [comment]
Testing http API with curl in dev environment.

Снимок экрана 2023-08-28 в 15 31 26

File: modules/002-deckhouse/templates/deployment.yaml (hunk 1 above)
Comment (Dmitrykob) [reply modules/002-deckhouse/templates/deployment.yaml]: Done

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by yalosev — APPROVED** [review]

**Review by name212 — APPROVED** [review]

**Pull request merged**